static/app/views/nav/primary/components.tsx

```diff
@@ -329,16 +493,34 @@ export const SidebarItemUnreadIndicator = styled('span')`
   border: 2px solid ${p => p.theme.background};
 `;
 
-const SeparatorListItem = styled('li')`
+export const SidebarList = styled('ul')<{isMobile: boolean; compact?: boolean}>`
+  position: relative;
   list-style: none;
+  margin: 0;
+  padding: 0;
+  padding-top: ${space(1)};
+  display: flex;
+  flex-direction: column;
+  align-items: ${p => (p.isMobile ? 'stretch' : 'center')};
+  gap: ${space(0.5)};
   width: 100%;
-  padding: 0 ${space(1.5)};
+
+  /* TriggerWrap div is getting in the way here */
+  ${p =>
+    p.theme.isChonk &&
+    css`
+      > div,
+      > li {
+        width: 100%;
+      }
```

Comment:
the coupling to dom structure is pretty rough here 😢

Reply:
Yeah, I don't like this. I added both because I assume the intermediary div will be removed at some point because it's also resulting in ul > div html structure